pupil_src/shared_modules/calibration_choreography/single_marker_plugin.py

```diff
@@ -56,7 +56,9 @@ class SingleMarkerMode(enum.Enum):
 
     @staticmethod
     def all_modes() -> T.List["SingleMarkerMode"]:
-        return list(SingleMarkerMode)
+        modes = list(SingleMarkerMode)
+        modes = sorted(modes, key=lambda m: m.order)
+        return list(modes)
```

Comment:
`sorted()` already returns a list.

Maybe we could even
```python
return sorted(SingleMarkerMode, key=lambda m: m.order)
```
And when I think about it more: why not just define them in the right order?
Then you can also remove the `order()` property below.
Then you can also remove this method here and just iterate the enum in the places where needed.

Reply:
I like that this is implemented explicitly. I would not want the UI order to be dependent on the enum definition order.

Reply:
Ok I guess that's a question of taste :slightly_smiling_face: 
Still, my first argument about `sorted` holds.